pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -884,15 +884,28 @@ void splitAndOwnBundleOnceAndRetry(NamespaceBundle bundle,
                                             .thenRun(() -> {
                                                 bundleFactory.invalidateBundleCache(bundle.getNamespaceObject());
                                                 updateFuture.complete(splittedBundles.getRight());
```

Comment:
It seems `updateFuture` should complete after `updateNamespaceBundlesForPolicies` completes.

Reply:
Fixed,PTAL,thanks! @Jason918